notes/src/components/NoteViewPage.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import Navigation from './Navigation';
+import NoteView from './NoteView';
+import Modal from './Modal';
+
+import './NoteViewPage.css';
+
+class NoteViewPage extends Component {
+
+  componentDidMount() {
```

Comment:
I like that you're thinking about the usability of the site here. Have you considered having the `<Navigation>` component `position: fixed` in mobile view?

Reply:
Makes perfect logical sense! Thanks for mentioning it.